source/usage-examples/count.txt

```diff
@@ -33,14 +33,17 @@ accepts a :doc:`query </fundamentals/crud/query-document>` parameter.
 settings that affect the method's execution. Refer to the reference
 documentation for each method for more information.
 
-.. important::
+.. tip::
 
-   If you require an exact document count in your collection, use a query
-   to take advantage of the built-in index on the ``_id`` field.
+   You can avoid a collection scan when using ``countDocuments()`` to return
+   the total number of documents in a collection by using a :manual:`hint
+   </reference/method/cursor.hint>` to take advantage of the built-in index on
+   the ``_id`` field. Use this technique only when calling ``countDocuments()``
```

Comment:
[suggestion]
Consider breaking this up into two sentences to make it easier to parse. It took me a couple of reads to absorb what it was saying because of it's length.

For example:
You can improve the performance when using ``countDocuments()`` to return the total number of documents by avoiding a collection scan. To do this, use a `hint` to take advantage of the built-in index for the ``_id`` field."